tests/plan/show/test.sh

```diff
@@ -28,19 +28,19 @@ rlJournalStart
         rlAssertGrep "relates https://something.org/related" $rlRun_LOG
 
         # Steps
-        rlRun "grep -A2 '^ *discover' $rlRun_LOG > $output"
+        rlRun "grep -Pz '(?sm)^ *discover ?$.*^ *provision' $rlRun_LOG > $output"
```

Comment:
How is this supposed to work? Before it provided short context after `discover`:

```
    discover 
         how fmf
dist-git-remove-fmf-root false
```

The fresh regexp seems to show the full plan:

```
> tmt plan show full | grep -Pz '(?sm)^ *discover ?$.*^ *provision'
/plans/full
     summary Plan keys are correctly displayed
 description Some description
    discover 
         how fmf
        test 
      filter tier:1
        link 
     exclude 
dist-git-merge false
dist-git-remove-fmf-root false
dist-git-init false
modified-only false
      fmf-id false
dist-git-source false
   provision 
         how container
       image fedora
  force-pull false
        user root
     prepare 
         how shell
      script systemctl start libvirtd
      report 
         how html
        open true
      finish 
         how ansible
    playbook cleanup.yaml
 environment KEY: VAL
     context distro: fedora
     enabled true
       order 70
          id e3a9a8ed-4585-4e86-80e8-1d99eb5345a9
         tag foo
        tier 3
     relates https://something.org/related
```

Reply:
The original test was expecting `filter` to be somewhere within 2 lines after `discovery`, which is no longer true, thanks to the increased verbosity of `show` (temporary, hopefully, as discussed on IRC). Therefore I changed the pattern to not output 2 lines and nothing more, but rather look for an ending delimiter, and not `grep` prints all lines from `discover` till `provision`, and `filter` should be somewhere in this pile for sure.

Reply:
The pattern works, but it's missing `-o` aka "print only matched". Will fix.

Reply:
Added a fix in [8d096fd](https://github.com/teemtee/tmt/pull/1461/commits/8d096fd6dedfb1f4dab9cea49f75946075af5836)